docs/caveat-unique-values.doctest.rst.inc

```diff
@@ -0,0 +1,121 @@
+.. _caveat-unique-values:
+
+Values Must Be Unique
+---------------------
+
+Consider the following::
+
+    >>> from bidict import bidict
+    >>> b = bidict({0: 0, 1: 1})
+    >>> b[0] = 1  # doctest: +SKIP
+
+What should happen next?
+
+If the bidict allowed this to succeed,
+because of the uniqueness-of-values constraint,
+it would silently clobber both of the existing mappings,
+possibly unbeknownst to you,
+resulting in surprises or problems down the line.
+
+Instead, bidict raises an exception
+any time you attempt to insert a non-unique value,
+so you have an opportunity to catch this early
+and resolve the conflict before it causes problems later on::
+
+    >>> b = bidict({1: 1})
+    >>> b[2] = 1
+    Traceback (most recent call last):
+        ...
+    ValueExistsException: (1, 1)
```

Comment:
Wondering whether the `__str__()` for ValueExistsException could be a little less cryptic, eg

    def __str__(self):
        return 'Value {1!r} exists for key {0!r}'.format(*self.args[0])

This would naturally only have a cost if the exception needed to be rendered as a string.